docs/Using-Fleet/MDM-setup.md

```diff
@@ -221,42 +232,43 @@ All automatically-enrolled hosts will be assigned to a default team of your choo
 
 > A host can be transferred to a new (not default) team before it enrolls. Learn how [here](./Teams.md#transfer-hosts-to-a-team). Transferring a host will automatically enforce the new team's settings when it enrolls.
 
-To change the default team, choose the "Fleet UI" or "fleetctl" method and follow the steps below.
+Use either of the following methods to change the default team:
 
-Fleet UI:
+#### Fleet UI
 
-1. In the Fleet UI, head to the **Settings > Integrations > Mobile device management (MDM)** page.
+1. Navigate to the **Settings > Integrations > Mobile device management (MDM)** page.
 
 2. In the Apple Business Manager section, select the **Edit team** button next to **Default team**.
 
 3. Choose a team and select **Save**.
 
-`fleetctl` CLI:
+#### Fleetctl CLI
 
 1. Create a `config` YAML document if you don't have one already. Learn how [here](./configuration-files/README.md#organization-settings). This document is used to change settings in Fleet.
 
 2. Set the `mdm.apple_bm_default_team` configuration option to the desired team's name.
 
 3. Run the `fleetctl apply -f <your-YAML-file-here>` command.
 
-### Pending hosts
-Some time after you purchase a Mac through Apple or an authorized reseller, but before it has been set up, the Mac will appear in ABM as in transit. When the Mac appears in ABM, it will also appear in Fleet with **MDM status** set to "Pending." After the new host is set up, the **MDM Status** will change to "On" and the host will be assigned to the default team.
+>**Pending hosts**. Some time after you purchase a Mac through Apple or an authorized reseller, but before it has been set up, the Mac will appear in ABM as in transit. When the Mac appears in ABM, it will also appear in Fleet with **MDM status** set to "Pending." After the new host is set up, the **MDM Status** will change to "On" and the host will be assigned to the default team.
```

Comment:
```suggestion
>### Pending hosts 
Some time after you purchase a Mac through Apple or an authorized reseller, but before it has been set up, the Mac will appear in ABM as in transit. When the Mac appears in ABM, it will also appear in Fleet with **MDM status** set to "Pending." After the new host is set up, the **MDM Status** will change to "On" and the host will be assigned to the default team.
```